examples/relay-server/README.md

```diff
@@ -0,0 +1,28 @@
+## Description
```

Comment:
We might want to move this one into the dcutr example unless we need it in a different one as well?

Reply:
@thomaseizinger I left it separate as it was an example on its own. And I put the `relay-server` folder link in the `dcutr` example. This way the user reading the `dcutr` example can directly look in the `relay-server` folder. Another user just wanting to learn how a `relay-server` works can run the full-fledged `relay-server` example and play with it. What do you think?